README.md

```
@@ -183,14 +183,42 @@ The `./scripts/run_cloudxr.sh` script will automatically:
+`https://nvidia.github.io/IsaacTeleop/client/`
+
+In that page, set **Server IP** to your server IP (for example, `192.168.1.2`).
+
 7. **White list ports for Firewall**
 
-Open the [required CloudXR ports](https://docs.nvidia.com/cloudxr-sdk/latest/requirement/network_setup.html#ports-and-firewalls)
-and the web server ports for the WebXR client:
+Open the [required CloudXR ports](https://docs.nvidia.com/cloudxr-sdk/latest/requirement/network_setup.html#ports-and-firewalls).
+
+If you are running the web client on this machine:
 ```bash
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)